spec/policy_machine_storage_adapters/active_record_spec.rb

```diff
@@ -25,6 +25,328 @@
     it_behaves_like 'a policy machine storage adapter'
     let(:policy_machine_storage_adapter) { described_class.new }
 
+    describe 'scoping privileges by user attribute' do
+      let(:candy_pm) { PolicyMachine.new(name: 'Candy Machine', storage_adapter: PolicyMachineStorageAdapter::ActiveRecord)}
```

Comment:
Resident unfun person checking in for duty – I don't think fun names make tests any clearer (this is a recent change for me). I think they obfuscate and distract from the specific things that are actually being tested.

I'm sympathetic to the argument that more human names for variables and objects can paint a picture that's easier for some people to follow, but on the whole, I think they engender more confusion than they mitigate.

In any case, this is a personal preference of mine that I just wanted to call out. I don't expect you to change it on this PR.

Reply:
Fair enough, although I think there's a difference between "fun names" and "more human names." Fun names can be distracting, but more human names significantly help me process the graph relationships going on in a lot of these specs.

That being said, I definitely agree that human readable names can easily slip into obfuscation. I'll cool it in future specs.